zipline/algorithm.py

```diff
@@ -1519,25 +1971,33 @@ def set_max_order_size(self, asset=None, max_shares=None,
 
     @api_method
     def set_max_order_count(self, max_count):
-        """
-        Set a limit on the number of orders that can be placed within the given
-        time interval.
+        """Set a limit on the number of orders that can be placed in a single
+        day.
+
+        Parameters
+        ----------
+        max_count : int
+            The maximum number of orders that can be placed on any single day.
         """
         control = MaxOrderCount(max_count)
         self.register_trading_control(control)
 
     @api_method
     def set_do_not_order_list(self, restricted_list):
-        """
-        Set a restriction on which assets can be ordered.
+        """Set a restriction on which assets can be ordered.
+
+        Parameters
+        ----------
+        restricted_list : set[Asset]
```

Comment:
I think set connotes that it is an unordered container. I could change this to `container[Asset]`